dist/templates/ovn-ipsec.yaml.j2

```diff
@@ -101,7 +102,6 @@ spec:
             # Generate an SSL private key and use the key to create a certitificate signing request
             umask 077 && openssl genrsa -out /etc/openvswitch/keys/ipsec-privkey.pem 2048
             openssl req -new -text \
-                        -extensions v3_req \
```

Comment:
Can you explain this ?

Reply:
as explained in the commit message, CSR creation is failed with `-extensions v3_req` parameter on fedora, so had to remove this, but this didn't harm in configuring IPsec with libreswan.

Reply:
I am not a SME on this so i cannot say if this is ok or not.. if its not. Id add it to a new PR if its unrelated.

Reply:
can we get some security specialist to review this? 
I cannot validate removing this arg. :/ 
I dont have experience so I cannot confirm if what you said is true.

Reply:
this should be seperate PR i think - seems unrelated to mertrics.